code/dynamic_programming/src/LongestRepeatingSubsequence.cpp

```diff
@@ -0,0 +1,56 @@
+//dynamic programming || Longest repeating subsequence
+
+#include <iostream>
+#include<cstring>
+using namespace std;
+
+int longest_repeating_subsequence(string s)
+{
+    int n=s.size();  // Obtaining the length of the string
+    int dp[n+1][n+1];
+
+    //Initializing the table
+
+    for (int x=0;x<=n;x++)
+    {
+        for (int y=0;y<=n;y++)
+        {
+            dp[x][y]=0;
+        }
+    }
+
+    // Implementation is very similar to Longest Common Subsequence problem
+
+    for (int x=1;x<=n;x++)

```

Comment:
All of your expressions, including variables, conditionals, and loops need spacing between operators.